### ISSUE TITLE
Análisis de Viabilidad Técnica para desarrollo de Reportes Dinámicos

### ISSUE THREAD

**Issue by AlexFBP** [udistrital/arka_cliente#724]
En la mesa de trabajo del 27/07/21 (y en otras anteriores) se mencionó si había la posibilidad de realizar reportes dinámicos: Que se puedan seleccionar campos arbitrariamente para generar reportes.

**Comment by mcrubianot** [comment]
Efectivamente, los requerimientos que se han hecho con respecto a reporte, lo que indican es que se puedan generar los reportes por varias extensiones, pdf, excel, csv, htm, Lo anterior teniendo en cuenta que no todos lo repores son iguales y muchas veces lo que han requerido a la OAS es que se les construya, por que los sistemas no los tienen.

Para el caso de los reportes propios de los documentos si es necesario revisar la estructura de cada uno de ellos. 

Les comparto un documento inicial que se levantó, donde se muestran algunas estructuras.

https://docs.google.com/spreadsheets/d/1-OIK5CRvH106cvc5DYgJcTqxn-ldFMiDE-PnnBxTRaM/edit?usp=sharing

Bueno, ingenieros hay que tener en cuenta los reportes que se deben generar para la Secretaría de Hacienda - Sistema Bogdata, donde los lineamientos de los reporte de ellos si son muy puntuales y no pueden ser modificables. Considero pertinente revisar el tema con la persona que maneja Sicapital en la OAS, ya que desde allí, creo ya están generando algunos.

Quedo atenta a cualquier comentario al respecto.
